src/types/ajv/CycleRecordSchema.ts

```diff
@@ -0,0 +1,345 @@
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          cycleJoined: { type: 'string' },
+          counterRefreshed: { type: 'number' },
+          id: { type: 'string' },
+        },
+      },
+    },
+    leavingArchivers: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          publicKey: { type: 'string' },
+          ip: { type: 'string' },
+          port: { type: 'number' },
+          curvePk: { type: 'string' },
+        },
+        required: ['publicKey', 'ip', 'port', 'curvePk'],
+      },
+    },
+    lost: { type: 'array', items: { type: 'string' } },
+    lostAfterSelection: { type: 'array', items: { type: 'string' } },
+    lostArchivers: { type: 'array', items: { type: 'string' } },
+    lostSyncing: { type: 'array', items: { type: 'string' } },
+    marker: { type: 'string' },
+    maxSyncTime: { type: 'number' },
+    nodeListHash: { type: 'string' },
+    refuted: { type: 'array', items: { type: 'string' } },
+    refutedArchivers: { type: 'array', items: { type: 'string' } },
+    removed: { type: 'array', items: { type: 'string' } },
+    removedArchivers: { type: 'array', items: { type: 'string' } },
+    returned: { type: 'array', items: { type: 'string' } },
+    standby: { type: 'number' },
+    standbyAdd: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          appJoinData: {
+            type: 'object',
+            properties: {
+              adminCert: { type: ['null', 'string'] },
+              mustUseAdminCert: { type: 'boolean' },
+              stakeCert: {
+                type: ['object', 'null'],
+                properties: {
+                  certExp: { type: 'number' },
+                  nominator: { type: 'string' },
+                  nominee: { type: 'string' },
+                  signs: {
+                    type: 'array',
+                    items: {
+                      type: 'object',
+                      properties: {
+                        owner: { type: 'string' },
+                        sig: { type: 'string' },
+                      },
+                      required: ['owner', 'sig'],
+                    },
+                  },
+                  stake: {
+                    type: 'object',
+                    properties: {
+                      dataType: { type: 'string' },
+                      value: { type: 'string' },
+                    },
+                    required: ['dataType', 'value'],
+                  },
+                },
+                required: ['certExp', 'nominator', 'nominee', 'signs', 'stake'],
+              },
+              version: { type: 'string' },
+            },
+            required: ['mustUseAdminCert', 'stakeCert', 'version'],
+          },
+          cycleMarker: { type: 'string' },
+          nodeInfo: {
+            type: 'object',
+            properties: {
+              publicKey: { type: 'string' },
```

Comment:
node info is missing fields .. activeTimestamp, syncingTimestamp, refreshed Counter is also part of node info but not required.. please use the below for reference : 
![image](https://github.com/user-attachments/assets/2a27f5d2-0b53-42ce-9307-dd2c6c14a0c2)